src/mqt/core/_core/__init__.pyi

```diff
@@ -217,6 +217,9 @@ class QuantumComputation(MutableSequence[Operation]):
     def iswap(self: Self, target1: int, target2: int) -> None: ...
     def ciswap(self: Self, control: Control, target1: int, target2: int) -> None: ...
     def mciswap(self: Self, controls: set[Control], target1: int, target2: int) -> None: ...
+    # def iswapdg(self: Self, target1: int, target2: int) -> None: ...
+    # def ciswapdg(self: Self, control: Control, target1: int, target2: int) -> None: ...
+    # def mciswapdg(self: Self, controls: set[Control], target1: int, target2: int) -> None: ...
```

Comment:
These should not be commented. The library exposes them, so hey should be in the stub file.

Reply:
Done ✅